src/iop/useless.c

```diff
@@ -21,18 +21,37 @@
 // our includes go first:
 #include "bauhaus/bauhaus.h"
 #include "develop/imageop.h"
+#include "develop/imageop_gui.h"
 #include "gui/gtk.h"
 #include "iop/iop_api.h"
 
 #include <gtk/gtk.h>
 #include <stdlib.h>
 
+// This is an example implementation of an image operation module that does nothing usefull.
+// It demonstrates how the different functions work together. To build your own module,
+// take only the required functions, stripping them of comments, and add the ones to implement
+// the functionality you need. Don't copy default implementations (hint: if you don't need to change
+// anything, you probably don't need the copy)
+
 // this is the version of the modules parameters,
```

Comment:
(not your comment I know but) should be "module parameters"

Reply:
or maybe even "module's parameters"?

Reply:
That works too